modules/assim.sequential/R/sda.enkf.R

```diff
@@ -929,7 +1001,9 @@ sda.enkf <- function(settings, obs.mean, obs.cov, IC = NULL, Q = NULL, adjustmen
     ###-------------------------------------------------------------------###
     ### save outputs                                                      ###
     ###-------------------------------------------------------------------### 
-    save(t, FORECAST, ANALYSIS, enkf.params, file = file.path(settings$outdir, "sda.output.Rdata"))
+    #save(t, FORECAST, ANALYSIS, enkf.params, file = file.path(settings$outdir, "sda.output.Rdata"))
+    save(list = ls(envir = environment(), all.names = TRUE), 
```

Comment:
This could get kind of huge right? Do we need the whole environment?

Reply:
yeah, I was only working with ensemble kalman filter so it wasn't getting as big as your. I changed this for development, will change it back